api/src/pcapi/domain/payments.py

```diff
@@ -354,3 +357,18 @@ def _get_booking_type(booking: Booking) -> str:
         return "PC"
 
     return "UNKNOWN"
+
+
+def _get_booking_ministry(booking: Booking) -> str:
+    if booking.educationalBooking is not None:
+        deposits = booking.educationalBooking.educationalInstitution.deposits
+
+        if len(deposits) > 0:
+            # deposits are filtered in query so that only the deposit associated with the booking is kept
+            ministry = booking.educationalBooking.educationalInstitution.deposits[0].ministry
+
+            # todo: delete this condition when data is populated with ministry
+            if ministry is not None:
+                return ministry.name
+
+    return ""
```

Comment:
Early retturn pattern pour éviter les if nestés

```suggestion
    if booking.educationalBooking is None:
        return ""
    
    deposits = booking.educationalBooking.educationalInstitution.deposits
    # deposits are filtered in query so that only the deposit associated with the booking is kept
    if len(deposits) != 1:
        logger.error('We should not have many or no deposit found [...]' (je te laisse voir pour ajouter des infos dans les extras))
        return ""
    
    ministry = booking.educationalBooking.educationalInstitution.deposits[0].ministry
    # todo: delete this condition when data is populated with ministry
    if ministry is None:
        return ""

    return ministry.name
```

Reply:
ouais je trouvais ca moche d'avoir plusieurs fois le même return ^^

Reply:
Tu peux définir une valeur par défaut, mais pour un "" je trouve pas ça gênant :) 

Visuellement c'est + sympa d'avoir un seul return mais conceptuellement c'est + compliqué de comprendre les différentes conditions qui mènent au ministry.name alors qu'en faisant du early return tu fais apparaître les conditions de manière explicite